modules/nf-core/limma/differential/tests/nextflow.interaction.config

```diff
@@ -0,0 +1,13 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: 'LIMMA_DIFFERENTIAL' {
+        ext.args = { [
+            "--sample_id_col sample",
+            "--probe_id_col gene_id",
+            "--blocking_variables $meta.blocking"
```

Comment:
why is blocking variables specified here? I thought this is not used when a formula is specified?

Reply:
It is not used, indeed. Leftover from other config. I should remove this.